x-pack/plugins/ml/server/lib/alerts/register_anomaly_detection_alert_type.ts

```diff
@@ -42,10 +48,51 @@ export type AnomalyDetectionAlertContext = AnomalyDetectionAlertBaseContext & {
   topInfluencers?: InfluencerAnomalyAlertDoc[];
 };
 
+export type ExecutorOptions<P extends RuleTypeParams> = RuleExecutorOptions<
+  P,
+  RuleTypeState,
+  {},
+  AnomalyDetectionAlertContext,
+  typeof ANOMALY_SCORE_MATCH_GROUP_ID,
+  MlAnomalyDetectionAlert
+>;
+
 export const ANOMALY_SCORE_MATCH_GROUP_ID = 'anomaly_score_match';
 
 export type AnomalyScoreMatchGroupId = typeof ANOMALY_SCORE_MATCH_GROUP_ID;
 
+export const ANOMALY_DETECTION_AAD_INDEX_NAME = 'ml.anomaly-detection';
+
+const ML_ALERT_NAMESPACE = ALERT_NAMESPACE;
+
+export const ALERT_ANOMALY_DETECTION_JOB_ID = `${ML_ALERT_NAMESPACE}.jobId` as const;
```

Comment:
Should `job_id` be an array in case we add support for multiple jobs in the anomaly detection rule?

And if we used the same index for the jobs health rule, would we need this field to be an array as that rule does allow you to configure multiple jobs?

Reply:
renamed in https://github.com/elastic/kibana/pull/166349/commits/db719c98b46cd0b1e650bfac33a691db4c44993a

Reply:
> And if we used the same index for the jobs health rule, would we need this field to be an array as that rule does allow you to configure multiple jobs?

No, because we allow multiple jobs only for the rule config. Alerts are created **per job** for both rule types.